### PR TITLE
docs: fix spacing and typography 

### LINKED ISSUE #919: [docs] Spacing and typography style fixes
- [x] `h3` font size should be 20px
- [x] Spacing between `h2` and `h3` should be 32px

- [x] `h3` spacing should be 16px between body copy and 32px between alerts. Spacing should be 16px between `h3` and images/example containers.
Screenshot 2023-04-13 at 10 59 34 AM

- [x] Example containers should have border radius of 8px.
- [x] Example containers should have 32px padding for small screens. Examples with darkest palette has 64px padding at any screen size.

- [x] Font size of body copy on homepage is 16px.
Screenshot 2023-04-13 at 12 34 21 PM

- [x] Page headers should have 80px padding-top for large screen sizes
Screenshot 2023-04-13 at 12 42 32 PM

### PR DESCRIPTION
**Pull request by marionnegp**
## What I did

I changed spacing between headings, body paragraphs, and other layout elements. Some of the spacing will still look funky until the proper `<section>` tags and heading levels are implemented or cleaned up on individual element and pattern pages.

## Testing Instructions

In the deploy preview:
- The content on the Accordion and CTA pages have good examples to check spacing and heading styles.
- Check the homepage to make sure body copy is the right size.
- Check that the border radius for cards or example containers are 8px.

## Notes to Reviewers
CLoses #919

### CODE REVIEW

**Comment by marionnegp** [comment]
@coreyvickery, this is the PR I'm using to make typography and spacing updates related to #919. It's in draft until we decide on what size to make `<h4>` if our `<h3>` needs to be 20px.

File: docs/scss/_base/_base.scss [hunk 1]
```diff
@@ -57,7 +57,7 @@ h2,
 h3,
 h4 {
   font-family: 'RedHatDisplay', 'Overpass', Helvetica, sans-serif;
-  line-height: 1.4;
+  line-height: 1.3;
```
Comment (bennypowers) [inline docs/scss/_base/_base.scss:60]: if we're anyways touching these rules can we advance to using tokens?

**Review by bennypowers — CHANGES_REQUESTED** [review]

File: docs/scss/_base/_base.scss (hunk 1 above)
Comment (marionnegp) [reply docs/scss/_base/_base.scss:60]: For sure. I'm hoping to get this PR merged, so our docs look more like our mockups asap. I've opened a new issue (#937) to switch over to tokens though.

**Review by coreyvickery — APPROVED** [review]
Looks good.

**Review by bennypowers — APPROVED** [review]

**Pull request merged**